api-reference/10 UI Components/dxScheduler/7 Interfaces/dxSchedulerToolbarItem/dxSchedulerToolbarItem.md

```diff
@@ -37,9 +37,7 @@ Predefined controls appear on the toolbar depending on whether a specific {Widge
 
     If you need to customize a predefined control, add an object to the **items[]** array. This object must contain the control's [name](/api-reference/10%20UI%20Components/dxScheduler/7%20Interfaces/dxSchedulerToolbarItem/name.md '/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/toolbar/items/#name') and [properties](/api-reference/10%20UI%20Components/dxScheduler/7%20Interfaces/dxSchedulerToolbarItem '/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/toolbar/items/') you want to customize. If a control does not need customization, you can include its name in the **toolbar**.**items[]** array. 
 
-    The date navigator has three components: left arrow (*'prev'*), right arrow (*'next'*), and date interval (*'dateInterval'*). You can customize the component order.
-
-    The example below adds a date navigator, and customizes the Today button:
+    The date navigator has three components: left arrow (*'prev'*), right arrow (*'next'*), and date interval (*'dateInterval'*). Define [options](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/toolbar/items/#options).**items** following the example below to customize the order of date navigator components.
```

Comment:
I think we should link to https://js.devexpress.com/jQuery/Documentation/ApiReference/UI_Components/dxButtonGroup/Configuration/ here. Because data navigator is a button group